examples/gno.land/r/demo/users/z_2_filetest.gno

```diff
@@ -1,6 +1,6 @@
 package main
 
-// SEND: 2000000000ugnot
+// SEND: 200000000ugnot
```

Comment:
Isn't it 20_000_000, instead of 200M?

Reply:
I think it should panic if any digit len error